### PR TITLE
Configurable pagination ref #821

### PR DESCRIPTION
**Pull request by leplatrem**
ref #821  (did not implement the «view all» idea)

Should we increase default to 50?

### CODE REVIEW

File: app/experimenter/settings.py [hunk 1]
```diff
@@ -34,6 +34,8 @@
 
 SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
 
+CSRF_TRUSTED_ORIGINS = [HOSTNAME]
```
Comment (jaredkerim) [inline app/experimenter/settings.py]: I think this belongs in a separate PR?

File: app/experimenter/settings.py [hunk 2]
```diff
@@ -62,7 +64,7 @@
     "django.middleware.security.SecurityMiddleware",
     "django.contrib.sessions.middleware.SessionMiddleware",
     "django.middleware.common.CommonMiddleware",
-    "django.middleware.csrf.CsrfViewMiddleware",
+    #    "django.middleware.csrf.CsrfViewMiddleware",
```
Comment (jaredkerim) [inline app/experimenter/settings.py]: Is this intentional? Or accidentally committed?

File: app/experimenter/settings.py [hunk 3]
```diff
@@ -173,7 +175,7 @@
     "loggers": {
         "django.db": {
             "handlers": ["console"] if DEBUG else [],
-            "level": "DEBUG",
+            "level": "WARNING",
```
Comment (jaredkerim) [inline app/experimenter/settings.py]: @peterbe Added a separate PR for making this more easily configurable, so you don't need to do this here.

File: docker-compose.yml [hunk 4]
```diff
@@ -37,8 +37,7 @@ services:
     links:
       - app
     ports:
-      - "80:80"
-      - "443:443"
+      - "4433:443"
```
Comment (jaredkerim) [inline docker-compose.yml]: I think you committed this by mistake?

**Comment by jaredkerim** [comment]
@leplatrem Other than all the stuff I think snuck into the commit by mistake, this looks really good!  Resubmit without that stuff and I'll merge :D

**Comment by leplatrem** [comment]
Ouuch, `git ci -am` is always at the tip of my fingers and I somehow comitted some temp changes I had in my workspace... Sorry for the noise!
